_docs_integrate/attribute-introduction.md

```diff
@@ -154,12 +154,12 @@ If an Identity is interested in Attributes from another Identity and wants to qu
 
 ### Create Attributes for peer
 
-Furthermore, it is possible to [create Attributes for another Identity]({% link _docs_integrate/create-attributes-for-peer.md %}). In this case, the sender of the Request determines the value of the Attributes. For example a school sends the students their certificates, then it has to use [Requests for creating Attributes]({% link _docs_integrate/create-attributes-for-peer.md %}#request-for-creating-attributes).
+Furthermore, it is possible to [create Attributes for another Identity]({% link _docs_integrate/create-attributes-for-peer.md %}). In this case, the sender of the Request determines the value of the Attributes. For example if a school sends the students their certificates, then it is necessary to use [Requests for creating Attributes]({% link _docs_integrate/create-attributes-for-peer.md %}#request-for-creating-attributes).
```

Comment:
For example if a school sends the students their certificates, (then) it **will be** necessary to use ...

Please mind the tense in if-sentences.